tests/fermi/test_fermionic.py

```diff
@@ -302,6 +516,34 @@ def test_copy(self, fs):
         assert copy_fs is not fs
         assert deep_copy_fs is not fs
 
+    def test_simplify(self):
+        """Test that simplify removes terms in the FermiSentence with coefficient less than the
+        threshold."""
+        un_simplified_fs = FermiSentence({fw1: 0.001, fw2: 0.05, fw3: 1})
+
+        expected_simplified_fs0 = FermiSentence({fw1: 0.001, fw2: 0.05, fw3: 1})
+        expected_simplified_fs1 = FermiSentence({fw2: 0.05, fw3: 1})
+        expected_simplified_fs2 = FermiSentence({fw3: 1})
+
+        un_simplified_fs.simplify()
+        assert un_simplified_fs == expected_simplified_fs0  # default tol = 1e-8
+        un_simplified_fs.simplify(tol=1e-2)
+        assert un_simplified_fs == expected_simplified_fs1
+        un_simplified_fs.simplify(tol=1e-1)
+        assert un_simplified_fs == expected_simplified_fs2
+
+    def test_pickling(self):
+        """Check that FermiSentences can be pickled and unpickled."""
+        f1 = FermiWord({(0, 0): "+", (1, 1): "-"})
+        f2 = FermiWord({(0, 0): "+", (1, 3): "-", (2, 0): "+", (3, 4): "-"})
+        fs = FermiSentence({f1: 1.5, f2: -0.5})
+
+        serialization = pickle.dumps(fs)
+        new_fs = pickle.loads(serialization)
+        assert fs == new_fs
```

Comment:
Are these new?

Reply:
No, but will confirm

Reply:
As above, not new, just shuffled around!